src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/appsettings.json

```diff
@@ -29,6 +29,7 @@
       {
         "Name": "File",
         "Args": {
+          // Replace App_Data with %ORCHARD_APP_DATA% to change log file location to use environment variable.
           "path": "App_Data/logs/orchard-log.txt",
```

Comment:
Should we also change `App_Data` with `%ORCHARD_APP_DATA%`? Then in the comment, we can explain what `%ORCHARD_APP_DATA%` does

Reply:
I did change - but then I reverted it - as in case of  default when `ORCHARD_APP_DATA` is not defined, it would change the current behavior, and would create logs in `ContentRoot\logs` instead of `ContentRoot\App_Data\logs`

Reply:
I think it is a very low risk breaking change that we can adapt. We should consider adding it and then document it in the 1.9 release as a breaking change.

Reply:
It's in template affects only new sites that's created with `dotnet new` - IMO we should not change  template with `%ORCHARD_APP_DATA% `